test/post-911-gib-status/components/PrintPage.unit.spec.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+
+import { PrintPage } from '../../../src/js/post-911-gib-status/containers/PrintPage';
+
+describe('<PrintPage/>', () => {
+  const pushSpy = sinon.spy();
```

Comment:
Just to be safe, what do you think of resetting this spy after each test?
```js
afterEach(() => pushSpy.reset());
```

Reply:
Ah! Good call.